src/Typeahead.svelte

```diff
@@ -174,15 +175,14 @@
   />
   {#if !hideDropdown && results.length > 0}
     <ul
-      class:svelte-typeahead-list={true}
```

Comment:
Could you restore the class name? It's a breaking change that's unrelated to the PR. I'd prefer to first patch the component before bumping a major version for this alone.